mkdocs.yml

```diff
@@ -64,6 +64,8 @@ extra:
       link: https://hub.docker.com/r/lmscommunity/lyrionmusicserver
     - icon: simple/vbulletin
       link: https://forums.slimdevices.com
+    - icon: octicons/graph-16
+      link: /analytics/
```

Comment:
This unfortunately opens the analytics in a new tab/window. Should we consider using the `copyright` configuration for some of these internal links? Downloads, analytics? It would render on the left hand side, on top of the "Made with...".

Reply:
I agree we should find a better solution for this, but I don't feel like using the copyright config is the right path. I'll look into extending the theme and overriding partials.

Reply:
I like to keep it simple!... would you try to override the new tab behaviour for the socials?

Reply:
I tried your suggestion using the copyright notice, and I must say it works good enough. So let's try that!